reactions.json

```diff
@@ -208,6 +208,34 @@
     "KI": "iodobenzene",
     "HBF4 / (Heat)": "fluorobenzene"
   },
+  "but-2-ene": {
+    "H2O / dil H2SO4": "butan-2-ol",
+    "PCC": "butan-2-one"
+  },
+  "propan-2-one": {

```

Comment:
Can you change it to propanone, the 2 is not needed.

Reply:
Definietely